modules/cluster-logging-curator-tolerations.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-curator.adoc
+
+[id="cluster-logging-curator-tolerations_{context}"]
+= Using tolerations to control the Curator pod placement
+
+You can control which node the Curator pod runs on
+using tolerations on the pod.
+
+You apply tolerations to the Curator pod through the Cluster Logging Custom Resource (CR)
+and apply taints to a node through the node specification. A taint on a node 
+instructs the node to repel all pods that do not tolerate the taint.
+
+
+.Prerequisites
+
+* Cluster logging and Elasticsearch must be installed.
+
+.Procedure
+
+. Use the following command to add a taint to a node where you want the Curator pod scheduled:
++
+----
+$ oc adm taint nodes <node-name> <key>=<value>:<effect>
+----
++
+For example:
++
+----
+$ oc adm taint nodes node1 key1=value1:NoExecute
+----
++
+This example places a taint on `node1` that has key `key1`, value `value1`, and taint effect `NoExecute`.
```

Comment:
Is there more information about available taints and acceptable key=value pairs? I see what the taint's doing, but IDK the significance of the key=value pair.

I might emphasize that you must use the NoExecute taint to make this work right.